packages/server/database/types/ImportedTask.ts

```diff
@@ -7,22 +6,23 @@ interface Input {
   meetingId: string
   teamId: string
   integration: TaskIntegration
+  integrationHash: string
   userId: string
 }
 
 export default class ImportedTask extends Task {
   integrationHash!: string
   integration!: TaskIntegration
   constructor(input: Input) {
-    const {meetingId, teamId, integration, userId} = input
+    const {meetingId, teamId, integration, userId, integrationHash} = input
```

Comment:
+1 can you give me some more info on why we can't use the `IntegrationHash.join` here?

Reply:
@mattkrick probably I had some issue in past which looks like not relevant anymore, reverted the change back